supervisor/dbus/timedate.py

```diff
@@ -66,20 +66,14 @@ async def connect(self):
             )
 
     @dbus_connected
-    def set_time(self, utc: datetime):
-        """Set time & date on host as UTC.
-
-        Return a coroutine.
-        """
-        return self.dbus.SetTime(int(utc.timestamp() * 1000000), False, False)
+    async def set_time(self, utc: datetime) -> None:
+        """Set time & date on host as UTC."""
+        await self.dbus.SetTime(int(utc.timestamp() * 1000000), False, False)
 
     @dbus_connected
-    def set_ntp(self, use_ntp: bool):
-        """Turn NTP on or off.
-
-        Return a coroutine.
-        """
-        return self.dbus.SetNTP(use_ntp)
+    async def set_ntp(self, use_ntp: bool) -> None:
+        """Turn NTP on or off."""
+        await self.dbus.SetNTP(use_ntp, False)
```

Comment:
I believe this function was actually broken. According to the [introspection](https://www.freedesktop.org/software/systemd/man/org.freedesktop.timedate1.html) the signature of `SetNTP` is this:
```
      SetNTP(in  b use_ntp,
             in  b interactive);
```
There are two parameters, not one. Calling it with one won't work.

We don't actually call this method anywhere in supervisor right now so that's probably how it slipped through.